eng/testing/WasmRunnerTemplate.cmd

```diff
@@ -56,11 +60,25 @@ if /I [%XHARNESS_COMMAND%] == [test] (
         )
     )
 ) else (
-    if [%BROWSER_PATH%] == [] if not [%HELIX_CORRELATION_PAYLOAD%] == [] (
-        set "BROWSER_PATH=--browser-path^=%HELIX_CORRELATION_PAYLOAD%\chrome-win\chrome.exe"
-    )
-    if [%JS_ENGINE_ARGS%] == [] (
-        set "JS_ENGINE_ARGS=--browser-arg^=--js-flags^=--stack-trace-limit^=1000"
+    if /I [%SCENARIO%] == [WasmTestOnChrome] (
+        if [%BROWSER_PATH%] == [] if not [%HELIX_CORRELATION_PAYLOAD%] == [] (
+            set "BROWSER_PATH=--browser-path^=%HELIX_CORRELATION_PAYLOAD%\chrome-win\chrome.exe"
+        )
+        if [%JS_ENGINE_ARGS%] == [] (
+            set "JS_ENGINE_ARGS=--browser-arg^=--js-flags^=--stack-trace-limit^=1000"
+        )
+    ) else (
+        if /I [%SCENARIO%] == [WasmTestOnFirefox] (
+            if [%BROWSER_PATH%] == [] if not [%HELIX_CORRELATION_PAYLOAD%] == [] (
+                set "BROWSER_PATH=--browser-path^=%HELIX_CORRELATION_PAYLOAD%\firefox\firefox.exe"
```

Comment:
nit: The chrome folder is `chrome-win`, can we have `firefox-win`? Or drop the suffix for both?

Reply:
this is inside `chrome-win.zip` and FF doesn't have it. There is also `chrome-linux` I'm not going to fiddle with that, sorry.

Reply:
Ook